multi/imxrt-multi/cm4.c

```diff
@@ -69,6 +70,10 @@ struct {
 	struct {
 		fifo_t rx;
 		fifo_t tx;
+		struct {
+			uint8_t busy : 1;
+			uint8_t nonblock : 1;
+		} state;
```

Comment:
As bitfields these might prove dangerous to use without a lock. Perhaps separate `volatile int`s?

Reply:
chanOpen() & chanClose() are protected by common mutex.

Reply:
Right, sorry